static/src/javascripts/projects/common/views/ui/notifications-subscribe-link.html

```diff
@@ -1,6 +1,6 @@
 <div class="js-notification-link notifications-link">
     <div class="notifications-subscribe-link--follow  <%= className %>">
-        <h3 class="notifications-subscribe-link__headline">Follow the story and stay updated</h3>
+        <div class="notifications-subscribe-link__headline">Follow the story and stay updated</div>
```

Comment:
I would say that a heading was the right mark-up semantically here, why did you change it to a div?